src/bufr_interface.h

```diff
@@ -372,6 +372,20 @@ extern "C" {
  */
   void get_inv_f(int lun, int** inv_ptr, int* inv_size);
 
+/**
+ *  Function used to get long strings from the BUFR file.
+ *
+ * @param lunit - Fortran logical unit.
+ * @param str_id - Mnemonic for the string for the source field plus the index number
+ *                  (ex: 'IDMN#2')
+ * @param output_str - The pre-allocated result string
+ * @param output_str_len - The length of the result string
+ *
+ * @author Ronald McLaren @date 2023-07-03
+ */
+
+  void readlc_f(int lun, const char* str_id, char* output_str, int* output_str_len);
+
```

Comment:
You've got a typo here which is causing the developer build to fail.   We've got WARN_AS_ERROR turned on now, which means if there's any mistake in the documentation (or if documentation is missing), then it's treated now as an error.

```
/home/runner/work/NCEPLIBS-bufr/NCEPLIBS-bufr/bufr/src/bufr_interface.h:378: error: argument 'lunit' of command @param is not found in the argument list of readlc_f(int lun, const char *str_id, char *output_str, int *output_str_len) (warning treated as error, aborting now)
```

It should actually be "int lunit" (not "int lun") in the function prototype.  lunit is the correct variable name in this case.